app/template_filters.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+
+
+def format_datetime(value: str) -> str:
```

Comment:
Use this in the browse.html template to format datestrings as formatted in the db to how we want them to be presnted in html